code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -56,30 +57,25 @@
 	if(tagged)
 		. += "bodybag_label"
 
-/obj/structure/closet/body_bag/open(mob/living/user, force = FALSE)
-	. = ..()
-	if(.)
-		mouse_drag_pointer = MOUSE_INACTIVE_POINTER
-
 /obj/structure/closet/body_bag/close()
 	. = ..()
 	if(.)
 		set_density(FALSE)
-		mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 
-/obj/structure/closet/body_bag/MouseDrop(over_object, src_location, over_location)
+/obj/structure/closet/body_bag/attack_hand_secondary(mob/user, list/modifiers)
 	. = ..()
-	if(over_object == usr && Adjacent(usr) && (in_range(src, usr) || usr.contents.Find(src)))
-		if(!attempt_fold(usr))
-			return
-		perform_fold(usr)
-		qdel(src)
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
+		return
+	if(!attempt_fold(user))
+		return SECONDARY_ATTACK_CONTINUE_CHAIN
+	perform_fold(user)
+	qdel(src)
```

Comment:
This means `..()` is being returned from `attack_hand_secondary`, which it never should be.